lib/helper/WebDriver.js

```diff
@@ -1312,6 +1312,17 @@ class WebDriver extends Helper {
     return this.browser.executeAsync.apply(this.browser, arguments);
   }
 
+  /**
+   * {{> scrollIntoView }}
+   *
+   */
+  async scrollIntoView(locator, scrollIntoViewOptions) {
```

Comment:
can we add tests @pablopaul?

Reply:
@PeterNgTr what would be your approach? Could you outline your steps for me? I try to follow them :)

Reply:
https://github.com/Codeception/CodeceptJS/blob/master/test/helper/WebDriver_test.js

so, I think you can add tests in here 👍

Reply:
@PeterNgTr I added a test